addons/hr_holidays/static/src/scss/accrual_plan_level.scss

```diff
@@ -55,36 +48,25 @@ $o-hr-holidays-border-color: map-get($grays, '300');
 
             // Whole record
             .o_hr_holidays_body {
-                margin-left: 8px;
-                padding-top: 20px;
 
                 // Left side 'Level'
                 .o_hr_holidays_timeline {
-                    @include o-position-absolute($top: 32px, $left: 6px);
                     width: 90px;
-                    padding: 3px 0px;
-                    border-radius: 3px;
-                    background-color: $o-btn-secondary-bg;
-                    box-shadow: 0 1px 2px rgba(0,0,0,.1);
                 }
 
                 // Actual kanban card
                 .o_hr_holidays_card {
-                    position: relative;
-                    margin-left: 22px;
-                    margin-right: 2px;
                     width: 500px;
-                    border-radius: 3px;
-                    background-color: $o-btn-secondary-bg;
-                    box-shadow: 0 1px 2px rgba(0,0,0,.1);
 
                     // Triangle
                     &:before {
                         content: '';
-                        @include o-position-absolute($top: 12px, $left: -17px);
-                        margin-left: 10px;
-                        width: 14px;
-                        height: 14px;
+                        position: absolute;
+                        top: map-get($spacers, 4);
+                        left: - map-get($spacers, 4);
+                        margin-left: $spacer;
```

Comment:
According to the css guidelines, the margin should be placed after the size properties.
I would also prefer to use `map-get($spacers, 3)`, it's better for maintenance (same for the other elements below).